keystone-init/README.md

```diff
@@ -174,6 +182,38 @@ containers:
 {{ include "keystone_env" .Values.my_pod.auth | indent(6) }}
 ```
 
+Endpoints are created by their unique name:
+```yaml
+endpoints:
+  example:
+    type:
+    region:
+    url:
+    description:
+    interfaces: []
+```
+In this case service with `example` name will be created first and then
```

Comment:
So ```example``` is not a name of an endpoint, but rather the name of service. If the service does not exists, it will be created prior to the endpoints creation ? 
Plus looking at other fields like ``type``, `region` it is more like creating a service and assigning the endpoints to it. 
Therefore I don't really like the ```endpoints``` as the way to have it inside the ```preload.yml```. I'd really go with ```services``` and have (optionally) ```interfaces``` changed to ```endpoints```. 

All that is based on a fact that you need a service to assign endpoints to, right ? Therefore the approach you took simse little bit unnatural :skull:

Reply:
@timothyb89 @matrixik told me that my idea is a bit different than the approach you initially had. You think my idea is valid or would you rather stay with the one that @matrixik has proposed ?

Reply:
I think I wrote the original keystone/preload.yml without a very good understanding of keystone's resource model :smiley: 

It looks like you're right, a [`Service`](https://docs.openstack.org/python-keystoneclient/latest/api/keystoneclient.v3.html#keystoneclient.v3.services.Service) contains some number of [`Endpoints`](https://docs.openstack.org/python-keystoneclient/latest/api/keystoneclient.v3.html#keystoneclient.v3.endpoints.Endpoint), and `interface` is just a property of `Service`. I'd be in favor of changing those names as you suggested.

Reply:
Done